### DISCUSSION TITLE
Не подключается к серверу(

### DISCUSSION THREAD

**Discussion by OneMoreAmnesia** [shadowsocks/shadowsocks-android#3183] [General]
Отвалился впн, говорит вот что
![Screenshot_20241015_232600_com github shadowsocks](https://github.com/user-attachments/assets/4ad2d59f-72c1-4deb-a15e-5894be13ab3d)

**Comment by dev4u** [comment]
建议用自建的服务，那个example的地址不清楚什么状况
